src/styles/components/_cookieBanner.scss

```diff
@@ -0,0 +1,54 @@
+.util-cookie-banner-container {
+  flex-direction: column;
+  border-width: 1px solid;
+  border-color: $black;
+  border-style: solid;
+  background-color: $white;
+  position: fixed;
+  max-width: 480px;
+  min-width: 480px;
+  bottom: 0;
+  left: 0;
+  height: 100%;
+  box-shadow: 0px 2px 10px -3px $x-d-grey;
+  font-size: 16px;
+  z-index: 1001;
+
+  &--show {
```

Comment:
should this be __show?

Reply:
> should this be __show?

My understanding from reading the BEM naming structure is that the double underscore means HAS. So hdr__list__item would mean that the header has a list that has an item. Then the double dash means an action. So hdr__list__item--show means the header has a list that has an item with an action of show on it. At least that's what I took from it when Xin went through this.
I just read Eamonn's link http://getbem.com/naming/ and it seems to indicate this. Let me know if you agree or not?